CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+All notable changes to this project 1.4.x per each release will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+
+## [v1.4.0.1] - 2019-11-28
+
+### Added
+- Introduce Helm charts for WSO2 Open Banking pattern 1
```

Comment:
@kaumini has to be `...Helm Chart...`.

Reply:
Fixed with 859a08f3bb7f9b8ec314bc9b4f1a9704e0b1324b